### ISSUE TITLE
After include statements the rest of the CSS is ignored when Scout-App processes my Sass

### ISSUE THREAD

**Issue by EgbertBleyenburg** [scout-app/scout-app#399]
There is something amis with my scout app. The following code 

```

@mixin background2x($path, $ext: "png", $w: auto, $h: auto, $pos: left top, $repeat: no-repeat) {

      $at1x_path: "#{$path}.#{$ext}";
      $at2x_path: "#{$path}@2x.#{$ext}";

      background-image: url("#{$at1x_path}");
      background-size: $w $h;
      background-position: $pos;
      background-repeat: $repeat;

      @media all and (-webkit-min-device-pixel-ratio : 1.5),
      all and (-o-min-device-pixel-ratio: 3/2),
      all and (min--moz-device-pixel-ratio: 1.5),
      all and (min-device-pixel-ratio: 1.5) {
        background-image: url("#{$at2x_path}"); 
      }
    }



    @media screen and (max-width: 768px){
          .navbar-toggle[aria-expanded="true"] {
             @include background2x("../images/navigation/menu-button-expanded",png,60px, 60px,left,no-repeat);
          }
          body {
             background:#ff0000;
          }
       }
```

The scout-app give me the following compiled code
```

@media screen and (max-width: 768px) {
  .navbar-toggle[aria-expanded=true] {
    background-image: url("../images/navigation/menu-button-expanded.png");
    background-size: 60px 60px;
    background-position: left;
    background-repeat: no-repeat;
  }

}
```

But is expected this

```
@media screen and (max-width: 768px) {
  .navbar-toggle[aria-expanded=true] {
    background-image: url("../images/navigation/menu-button-expanded.png");
    background-size: 60px 60px;
    background-position: left;
    background-repeat: no-repeat;
  }

  body {
    background: #ff0000;
  }
}
```

So, am i doing something wrong. Others parsers give me the correct code....

**Comment by TheJaredWilcurt** [comment]
This was a bug in the underlying Node-Sass engine that Scout-App uses. It has since been resolved and the latest Scout-App source code uses a newer version of Node-Sass. The next release of Scout-App will include this fix.

If you would like to use the fix now:

* https://github.com/scout-app/scout-app/blob/master/documentation/contributing.md#running-scout-app-locally-for-development

**Comment by TheJaredWilcurt** [comment]
@EgbertBleyenburg 

Closing issue. This has been released in **[Scout-App v2.18.16](https://github.com/scout-app/scout-app/releases/tag/v2.18.16)**.
